x/concentrated-liquidity/lp.go

```diff
@@ -171,6 +192,25 @@ func (k Keeper) withdrawPosition(ctx sdk.Context, owner sdk.AccAddress, position
 		if err := k.deletePosition(ctx, positionId, owner, position.PoolId); err != nil {
 			return sdk.Int{}, sdk.Int{}, err
 		}
+
+		hasPositionsRemaining, err := k.hasAnyPositionForPool(ctx, position.PoolId)
```

Comment:
Ah, I didn't understand what "hasAnyPositionForPool" meant until I saw it in this context. Maybe "anyPositionsRemainInPool" would be a better name, but no big deal if it stays this way